backends-common/rabbitmq/src/test/java/org/apache/james/backend/rabbitmq/RabbitMQExtension.java

```diff
@@ -71,24 +75,27 @@ public Object resolveParameter(ParameterContext parameterContext, ExtensionConte
         return rabbitMQ;
     }
 
-    public RabbitChannelPoolImpl getRabbitChannelPool() {
+    public RabbitChannelPoolImpl getRabbitChannelPoolImpl() {
```

Comment:
If the chammel pools are provided by the extension, then it should be closed by the extension, no?

Reply:
it is closed in `afterEach()`